src/python/pants/backend/javascript/package_json.py

```diff
@@ -707,7 +707,7 @@ async def generate_node_package_targets(
     package_target = NodePackageTarget(
         {
             **request.template,
-            NodePackageNameField.alias: pkg_json.name.replace("@", "__"),
+            NodePackageNameField.alias: pkg_json.name,
```

Comment:
Another oopsie, this has to actually be the correct string w.r.t what is in `package.json#name` field.